src/app_service/service/wallet_account/service.nim

```diff
@@ -416,7 +416,7 @@ QtObject:
 
   method toggleTestNetworksEnabled*(self: Service) =
     discard self.settingsService.toggleTestNetworksEnabled()
-    self.tokenService.init()
+    self.tokenService.loadData()
```

Comment:
You already have a `signalConnect` that does the same thing, so you can remove one of them

Reply:
nope the signal connect is when the wallet is enaled, this toggle is when test networks are enabled